HW8/OlenaBudko/Task_8_3/main.py

```
@@ -0,0 +1,22 @@
+from utils import *
+
+select_user = int(input("Enter 1- Area of rectangle, 2- Area of triangle or 3- Area of circle: "))
+if select_user == 1:
+    a = input("Enter side a of rectangle: ")
+    b = input("Enter side b of rectangle: ")
+    if a and b:
+        print(area_rectangle(float(a), float(b)))
```

Ruff (0.8.2)

`area_rectangle` may be undefined, or defined from star imports

(F405)